generators/client/templates/angular/webpack/_webpack.common.js

```diff
@@ -124,6 +125,15 @@ module.exports = function (options) {
                 $: "jquery",
                 jQuery: "jquery"
             }),
+            new MergeJsonWebpackPlugin({
+                output:{
+                    groupBy:[
+<%_ for (language in languages) { _%>
+                        { pattern:"./src/main/webapp/i18n/<%= languages[language] %>/*.json", fileName:"./src/main/webapp/i18n/<%= languages[language] %>/all.json" },
```

Comment:
Good question. My original thought was the same, however in that case, I couldn't figure out, how to run the server from my IDE, which is serving content from src/... I thought about adding a dynamic service, which could merge all the jsons automatically during runtime in dev mode. But it's seemed to be a bigger task than I would like to handle :smile:  
I've ended up commiting the merged jsons, which is sub-optimal solution.